### ISSUE TITLE
Invalid pinyin for 手 ( shǒu) ...

### ISSUE THREAD

**Issue by ogmartins** [tsroten/dragonmapper#7]
```python
import dragonmapper.hanzi as hz

In [1]: hz.to_pinyin('收')
Out[1]: 'shoū'

In [2]: hz.to_pinyin('手')
Out[2]: 'shoǔ'

In [62]: dragonmapper.__version__
Out[62]: '0.2.3'

python 3.4
```
Virtualenv in Ubuntu Linux Trusty

**Comment by tsroten** [comment]
@ogmartins Thank you for submitting this bug report. It looks like the accent being placed on the wrong value came from Unihan's data. I will correct the accent marks in the data and commit the new data file.

**Comment by tsroten** [comment]
I reported the error to the Unicode Consortium. Tomorrow, I'll correct the data in `dragonmapper`.

**Comment by ogmartins** [comment]
The problem is not the accent mark, but the *position* of the accent . This is
not standard pinyin.  CEDICT doest accept these entry: shoū  shoù  shoǔ.
```python
dragonmapper.transcriptions.is_pinyin('shoǔ')
False
dragonmapper.transcriptions.is_pinyin('shǒu')
True
dragonmapper.transcriptions.is_pinyin('shoù')
False
dragonmapper.transcriptions.is_pinyin('shoū ')
False

dragonmapper.transcriptions.pinyin_to_ipa('shoǔ')
ValueError: Not a valid syllable: o

dragonmapper.transcriptions.pinyin_to_ipa('shǒu')
'ʂoʊ˧˩˧'
```

**Comment by ogmartins** [comment]
These *non-standard* pinyn are  on the  lines from 24898 to 24904 of the file  hanzi_pinyin_characters.tsv.

**Comment by tsroten** [comment]
@ogmartins Yes, we're on the same page. I should have been clearer in my reply. The tone mark placement in the data file doesn't not follow [the Hanyu Pinyin rules](http://pinyin.info/rules/where.html), particularly with regard to `o` taking the mark in the `ou` vowel combination.

That data is taken directly from the `kHanyuPinlu` column of the Unihan Database files put out by the Unicode Consortium. I've contacted them and they are aware of the errors. As they will probably need quite a bit of time to sort it out on their end, I'll correct the errors on my end as well later today.

**Comment by tsroten** [comment]
@ogmartins Commit 81c90b7fe0a2780e1ad6ff582680abe2e3a03168 fixes 121 errors that were similar to the examples you gave above. Thanks again for pointing this out. A new PyPI release with this fix will be released shortly.

**Comment by tsroten** [comment]
I just released [version 0.2.4 to PyPI](https://pypi.python.org/pypi/dragonmapper/0.2.4), which includes the fix for this issue. Thanks again for your help.
